drools/src/main/java/com/datatorrent/drools/operator/StatelessDroolsOperator.java

```diff
@@ -140,4 +164,124 @@ public StatelessKieSession getKieSession()
   {
     return kieSession;
   }
+
+  private class RulesFiredListener implements AgendaEventListener
```

Comment:
can we extend DefaultAgendaEventListener, anyway we are not implementing most of the methods. Code will look more clean.